### ISSUE TITLE
Remote mutation on custom remote is broken

### ISSUE THREAD

**Issue by levitanong** [fulcrologic/fulcro-incubator#13]
Custom remotes don't seem to work. Seems ignored somewhere down the line.

## Minimal repro:
https://github.com/levitanong/repro-machine-remote

1. `shadow-cljs watch app`
2. visit `localhost:4000`

## Additional Information
```
VM25797:57 explcit remote is  :custom
goog.debug.console.js:203  [104.949s] [fulcro.client.impl.application] Use of invalid remote(s) detected!  #{:remote}
```

In `fulcro.client.impl.application/detect-errant-remotes` the deferred transactions (presumably related to pessimistic mutations) always use `:remote` rather than `custom`

**Comment by awkay** [comment]
Hm...there is something strange though

**Comment by awkay** [comment]
So, you're using the parallel parser, which returns an async channel.  Fulcro does not expect that to be the case, so it cannot properly block on your async logic.  The error message was incorrect, and I can correct that in incubator, but your example works if you use a parser that "blocks".

**Comment by awkay** [comment]
OK, with the delay it is behaving strangely still...I'll keep looking.

**Comment by awkay** [comment]
This isn't actually a incubator bug, it was a ptransact! bug in Fulcro.  I released a fix to Fulcro in 2.6.16.
